### ISSUE TITLE
Failed to activate the day-and-night package

### ISSUE THREAD

**Issue by theodorew1991** [jknlsn/day-and-night#4]
[Enter steps to reproduce:]

1. Open Atom.

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.4
**Thrown From**: [day-and-night](https://github.com/jknlsn/day-and-night) package 0.2.0

### Stack Trace

Failed to activate the day-and-night package

```
At _this.setup is not a function

TypeError: _this.setup is not a function
    at Object.activate (/packages/day-and-night/lib/day-and-night.js:52:11)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:253:1)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:225:1
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:99:1)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:218:1
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app/src/package.js:215:1)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:642:1)
    at /Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:519:1
    at exit (/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:128:1)
    at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:338:1)
    at /Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:351:1
    at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:220:1)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:493:12)
```

### Commands

```
     -0:39.5.0 core:select-up (input.hidden-input)
     -0:31.2.0 core:backspace (input.hidden-input)
     -0:29 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
asciidoc-preview 2.10.1 
atom-html-preview 0.1.22 
busy-signal 1.4.1 
copy-as-rtf 0.9.3 
day-and-night 0.2.0 
intentions 1.1.2 
language-latex 1.0.0 
language-matlab 0.2.1 
latex 0.43.0 
linter 2.1.4 
linter-gcc 0.7.1 
linter-pycodestyle 2.0.2 
linter-ui-default 1.6.0 
split-diff 1.4.0 
```

**Comment by theodorew1991** [comment]
This is a gamechanger

**Comment by jknlsn** [comment]
Fixed with version 0.2.1
